src/odin/enhancedtrippath.cc

```diff
@@ -1297,6 +1300,24 @@ bool EnhancedTripLeg_Node::HasIntersectingEdgeCurrNameConsistency() const {
   return false;
 }
 
+bool EnhancedTripLeg_Node::HasNonBackwardTraversableSameNameIntersectingEdge(
```

Comment:
Can you add some comments on what this function does?

Reply:
@mandeepsandhu i added comments